asm/dolphin/os/OSResetSW_data.s

```diff
@@ -2,24 +2,18 @@
 
 .section .sbss
 
-.global lbl_804D73A8
-lbl_804D73A8:
+.global ResetCallback
+ResetCallback:
 	.skip 0x4
-.global lbl_804D73AC
-lbl_804D73AC:
+.global Down
+Down:
```

Comment:
This is `static`, right?

Reply:
They'll all be static when I convert them to C in a subsequent PR

Reply:
Yeah, cool, that's what I meant.